source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+   located at the top of your dashboard. This brings up the pop-up dialog
+   box shown in figure 11 describing how to connect to your running instance. 
+
+.. figure:: figures/aws-web-11.png
+   :scale: 50 %
+   :alt: Connect to Your Instance
+
+   figure 11: :guilabel:`Connect to Your Instance`
+
+#. Open a terminal on your system. You should be in your :file:`$HOME`
+   directory.
+
+#. Copy the previously downloaded keyfile from the :file:`Downloads`
+   directory.
+
+   .. code-block:: console
+
+      cp Downloads/AWSClearTestKey.pem .
+
+#. Change the attributes of the :file:`AWSClearTestKey.pem` using the
+   :command:`chmod` command as instructed in the dialog box shown in figure
+   11.
+
+   .. code-block:: console
+
+      chmod 400 AWSClearTestKey.pem
+
+#. Copy the text highlighted in the :guilabel:`Example:` section that is
+   shown in figure 11. Paste the copied text into your terminal and add
+   `-l clear` text to the end of the command string. Press the :kbd:`Enter`
+   key to execute the command.
+
+   .. code-block:: console
+
+      ssh -i "AWSClearTestKey.pem" root@ec2-34-209-39-184.us-west-2.compute.amazonaws.com -l clear
+
+#. A message appears on the terminal stating the authenticty of the host can't
+   be established and prompts you with the message:
+
+   .. code-block:: console
+
+      The authenticity of host 'ec2-34-209-39-184.us-west-2.compute.amazonaws.com (34.209.39.184)' can't be established.
+      ECDSA key fingerprint is SHA256:LrziT5Ar66iBTfia8qmiIsrfBUm/UGam76U8bDR6yJc.
+      Are you sure you want to continue connecting (yes/no)?
+
+#. Type `yes` and hit the :kbd:`Enter` key. Another warning is printed to the
+   terminal and you are now at the command prompt of your new |CL| instance.
+
+   .. code-block:: console
+
+      Warning: Permanently added 'ec2-34-209-39-184.us-west-2.compute.amazonaws.com,34.209.39.184' (ECDSA) to the list of known hosts.
+      clear@clr-96a8565d0ca54b0c80364a1e5e7b0f88 ~ $ 
+
+Update the Clear Linux OS instance
+**********************************
+
+Run the :command:`sudo swupd update` command to update the operating
+system as shown in figure 12:
+
+.. figure:: figures/aws-web-12.png
+   :scale: 50 %
+   :alt: sudo swupd update
+
+   figure 12: :guilabel:`sudo swupd update`
+
+In this example we updated from version 18940 to 19100.
+
+Stop the Clear Linux OS instance
+********************************
+
+When you are finished using your AWS |CL| instance you need to stop it using
+the :guilabel:`Instances` dashboard to stop accruing charges. Complete the
+following steps from the :guilabel:`Instances` dashboard to stop your AWS |CL|
+instance from running.
+
+#. Select the :guilabel:`Actions` button to bring up a pull-down menu.
+
+#. Select the :guilabel:`Instance State` to expand the options.
+
+#. Select :guilabel:`Stop` to shut down the running instance.
+
+These steps are illustrated in figure 13.
```

Comment:
Passive voice: Figure 13 illustrates these steps. Be careful with the indentation. As is, the figure and this text break the numbering.

Reply:
done and good catch!